CHANGELOG.md

```diff
@@ -1,5 +1,86 @@
 [//]: # (current developments)
 
+## 24.1.0 (2024-01-24)
+
+### Enhancements
+
+* Verify signatures on to-be-installed packages instead of on all packages. (#11545, #13053)
+* Add a new `pre-solves` plugin hook. (#13053)
+* Add a new `post-solves` plugin hook. (#13053)
+* Add support for Python 3.12. (#13072)
+* Check `repodata.json.zst` for faster repodata downloads. (#13256)
+* Add `--skip-flexible-search` option to skip flexible search. (#13315)
```

Comment:
This is part of `conda search`, right? Might not be obvious to all readers. What about:

```suggestion
* Add `--skip-flexible-search` option in `conda search` to skip flexible search. (#13315)
```